src/components/Order/OrderMain/SelectedSeatList/SelectedSeatList.jsx

```diff
@@ -0,0 +1,27 @@
+import './SelectedSeatList.scss';
+
+const SelectedSeatList = ({ seatData, selected }) => {
+  const selectedSeat = (() => {
+    const chooseSeat = [];
+    selected.forEach((val, idx) => {
+      if (val) {
+        chooseSeat.push(idx);
+      }
+    });
+    return chooseSeat;
+  })(selected);
```

Comment:
- 이 즉시실행 함수에서, 인자를 받을 필요 없이 props로 받아온 값 자체를 사용하시면 됩니다(지금도 실제로 그렇게 사용하고 있고, 인자로 받아온 값을 사용하고 있지 않습니다).
- 즉시실행 함수 대신, array의 메서드를 활용하면 더 간편하게 만들어볼 수 있을 것 같습니다.
```suggestion
  const selectedSeats = seatData.map((el, idx) => el ? idx : null).filter(el => el !== null)
```